examples/data-objects/task-selection/tests/index.ts

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { getSessionStorageContainer } from "@fluid-experimental/get-container";
+import { IContainer } from "@fluidframework/container-definitions";
+
+import { oldestClientDiceId, taskManagerDiceId, TaskSelectionFactory } from "../src/containerCode";
+import { IDiceRoller } from "../src/interface";
+import { renderDiceRoller } from "../src/view";
+
+// Since this is a single page Fluid application we are generating a new document id
+// if one was not provided
+let createNew = false;
+if (window.location.hash.length === 0) {
+    createNew = true;
+    window.location.hash = Date.now().toString();
+}
+const documentId = window.location.hash.substring(1);
+
+async function requestDiceRoller(container: IContainer, id: string): Promise<IDiceRoller> {
```

Comment:
Can we replaced with requestFluidObject<IDiceRoller>(container, { url: id }).

Reply:
Will do.